compiler/rustc_parse/src/parser/pat.rs

```diff
@@ -133,7 +139,7 @@ impl<'a> Parser<'a> {
         };
 
         // Parse the first pattern (`p_0`).
-        let mut first_pat = self.parse_pat_no_top_alt(expected)?;
+        let mut first_pat = self.parse_pat_no_top_alt(expected, syntax_loc.clone())?;
```

Comment:
It's now `Copy`, right?
```suggestion
        let mut first_pat = self.parse_pat_no_top_alt(expected, syntax_loc)?;
```